views/group/new.pug

```diff
@@ -70,13 +68,12 @@ block content
           //- Heroicon name: x
           svg(xmlns='http://www.w3.org/2000/svg' alt="Bezárás" aria-label="Bezárás" fill='none' viewbox='0 0 24 24' stroke='currentColor' class='w-6 h-6')
             path(stroke-linecap='round' stroke-linejoin='round' stroke-width='2' d='M6 18L18 6M6 6l12 12')
-      //- TODO: side-calendar
-      //- div(class="flex flex-col h-full py-6 space-y-6 bg-white shadow-xl bg-default")
-      //-   header(class='px-4 sm:px-6')
-      //-     h2(class="text-lg font-medium leading-7") Szint foglaltsága
-      //-   div(class='relative flex-1 px-4 sm:px-6')
-      //-     div(class='absolute inset-0 h-screen px-4 sm:px-6')
-      //-       #side-calendar
+      div(class="flex flex-col h-full py-6 space-y-6 bg-white shadow-xl bg-default")
+        header(class='px-4 sm:px-6')
+          h2(class="text-lg font-medium leading-7") Szint foglaltsága
+        div(class='relative flex-1 px-4 sm:px-6')
+          div(class='absolute inset-0 h-screen px-4 sm:px-6')
+            #side-calendar
```

Comment:
The problem starts at line 65, not 68: This whole side panel thing is not so well formed and full of `position: absolute` positionings, I've got some solution for this, that I would happily tell you about (it is about flexbox :) )
I've contacted you on Slack we could have a meeting and solve this flexbox issue (and also take a look at the render problem).